gamemodes/terrortown/gamemode/server/sv_corpse.lua

```diff
@@ -235,19 +235,36 @@ local function ttt_call_detective(ply, cmd, args)
 
 	if IsValid(rag) and rag:GetPos():Distance(ply:GetPos()) < 128 then
 		if CORPSE.GetFound(rag, false) then
+			local plyTable = GetRoleChatFilter(ROLE_DETECTIVE, true)
+
+			hook.Run("TTT2ModifyCorpseCallRadarReceipients", plyTable, rag, ply)
+
 			-- show indicator in radar to detectives
 			net.Start("TTT_CorpseCall")
 			net.WriteVector(rag:GetPos())
-			net.Send(GetRoleChatFilter(ROLE_DETECTIVE, true))
+			net.Send(plyTable)
 
-			LANG.Msg("body_call", {player = ply:Nick(), victim = CORPSE.GetPlayerNick(rag, "someone")}, MSG_CHAT_PLAIN)
+			LANG.MsgAll("body_call", {player = ply:Nick(), victim = CORPSE.GetPlayerNick(rag, "someone")}, MSG_CHAT_PLAIN)
```

Comment:
But I don't like this change or the function at all, it's a useless function redirect, because https://github.com/TTT-2/TTT2/blob/01b27b75ef8b9eec8eeabdd5523ef23a0726b93f/gamemodes/terrortown/gamemode/shared/sh_lang.lua#L50 already does the same and looks a bit inconsistent.

Reply:
I mean, I can leave it. However do you see on first glance that `LANG.Msg` with three arguments and a stringa as the first argument means that it is broadcasted? Because I don't. And `LANG.MsgAll` clearly shows this.

Reply:
Then why not change `LANG.MsgAll` directly instead of broadcasting it with a string check inside the function / redirecting it? That's what I meant with 'inconsistency'

Reply:
I mean, I don't write that `LANG.Msg` function. It's ancient and is probably used by many addons. `Lang.MsgAll` calls `LANG.Msg` with `nil` as its first argument btw.

Reply:
Yep, that's a redirect because this function (A) is called, and calls another function (B) that doing a thing it would do even without calling function A.

Changing `MsgAll` into calling `LANG.ProcessMsg(nil, arg1, arg2, arg3)` directly would make more sense imo and should help to clear unnecessary nesting. But that's for another PR.